src/analytics/internal/analytics.js

```diff
@@ -54,7 +51,7 @@ class Analytics {
         }
 
         // Store the event in dexie
-        await this.storeEvent(eventArgs)
+        await window.eventLog.storeEvent(eventArgs)
```

Comment:
`window` should never be used for accessing global variables. I think the best thing here would be to allow your `EventLogBackground` instance to be passed into the `Analytics` instance via the `registerOperations` method (as this needs to be called before anything else works). Then set it as a prop on the instance and use that. (L54 + 67)